cap/modules/workflows/models.py

```diff
@@ -76,24 +78,27 @@ class ReanaWorkflow(db.Model):
         backref=db.backref('reana_workflows', cascade='all, delete-orphan')
     )
 
+    def __init__(self, **entries):
```

Comment:
why do we want to do this? 
this method can overwrite everything in the model, including even methods. Is it necessary?